src/controllers/health.controller.ts

```diff
@@ -0,0 +1,22 @@
+import type {
+  Handler,
+  Request,
+  Response,
+} from 'express';
+import type { HealthController } from '../types';
+import type { HealthStatus } from '../values';
+import { healthService } from '../services';
+
+const getHealth: Handler = (
+  req: Request,
+  res: Response,
+): void => {
+  const healthStatus: HealthStatus = healthService.getHealth();
+  res.json({
+    status: healthStatus,
+  });
```

Comment:
The call can be inlined:

```suggestion
  res.json({
    status: healthService.getHealth(),
  });
```